packages/adp-tooling/src/prompt/changes/prompts/change-data-source.ts

```diff
@@ -0,0 +1,298 @@
+import type { InputQuestion } from '@sap-ux/inquirer-common';
+import type { Answers, ListQuestion, Question, NumberQuestion } from 'inquirer';
+import type { AdpChangeDataSourceAnswers, AdpChangeDataSourceQuestions } from '../../../types';
+import { t } from '../../../i18n';
+import { validateURI, validateODataServices } from '../../validators';
+
+const ODATA_V2_PREFIX_ABAP = '/sap/opu/odata/';
```

Comment:
I don't understand what these prefixes do. Especially on an ABAP system I can define an alias for any of my ICF endpoints.

Reply:
Fixed with commit: fcb2eb3c603db4f005757221894faf5bf36e714e

Reply:
Looks much better however, there is still one usage at https://github.com/SAP/open-ux-tools/blob/fcb2eb3c603db4f005757221894faf5bf36e714e/packages/adp-tooling/src/prompts/change-data-source/index.ts#L70.

Reply:
This also no longer exists in the latest version of the code.
Fixed with: ecdfc9efae33793c05a54fe53f96a3921f703efd